### ISSUE TITLE
Multiple months

### ISSUE THREAD

**Issue by Drop-D** [jvolker/miq-nz-booking-tools#5]
Mr Volker thank you so much for doing this!

I now have it up and running and scanning October. I have no knowledge of Java and the way this script is working.

Can someone please advise if is it possible to scan more than one month, ie can I scan Nov and Oct?

Many thanks in advance

**Comment by shawry292** [comment]
Hi Drop-D, what was your line of code to add an additional month and scan for October too?

How did you approach this?

Thanks!

**Comment by Drop-D** [comment]
.
> Hi Drop-D, what was your line of code to add an additional month and scan for October too?
> 
> How did you approach this?
> 
> Thanks!

Sorry I was asking if it could be done. I can only check one month at a time.

**Comment by shawry292** [comment]
@Sandeep77R what exact .js file did you modify, and what line of code did you use to switch over or click to go to the next month. 

I tried `document.getElementsByClassName("pika-next").click()`

and 

`document.querySelector('.pika-next').click();`

or do we need something like: 

```
const next = document.querySelector('.pika-next');

function selectNextMonth() {
  console.log('The button was clicked.')
}

next.addEventListener('click', selectNextMonth);

next.click();
```

**Comment by Sandeep77R** [comment]
Hi - i'm an absolute beginner, so all I did was change the month from "5" to "11" in the code kindly written by Sir Volker, I assume Drop-D figured that out as well. I haven't been able to figure out how to scan two months at a go... 

I'm hardly getting any free spaces for group bookings however, well only seen 2 so far and both times its been for 10th July!

**Comment by shawry292** [comment]
Hi @Sandeep77R , you're referring to the index.js file and the MIQ Booking Assistance script yeah? Not the Google chrome extension right?

**Comment by Sandeep77R** [comment]
correct - the index.js file

**Comment by alexDrinkwater** [comment]
Fixed in #33
